ivadomed/loader/bids_dataframe.py

```diff
@@ -81,9 +81,9 @@ def create_bids_dataframe(self):
             # validate=True by default for both indexer and layout, BIDS-validator is not skipped
             # Force index for samples tsv and json files, and for subject subfolders containing microscopy files based on extensions.
             # Force index of subject subfolders containing CT-scan files under "anat" or "ct" folder based on extensions and modality suffix.
-            # TODO: remove force indexing of microscopy files after BEP microscopy is merged in BIDS
+            # TODO: remove force indexing of microscopy files after Microscopy-BIDS is integrated in pybids
             # TODO: remove force indexing of CT-scan files after BEP CT-scan is merged in BIDS
-            ext_microscopy = ('.png', '.tif', '.tiff', '.ome.tif', '.ome.tiff', '.ome.tf2', '.ome.tf8', '.ome.btf')
```

Comment:
Just curious, wasn't `tiff` the same as `tif` as equally valid extension? Or is it because we are doing partial matching and `.tif` can pickup both?

Reply:
You are right, `tiff` and `tif` are exactly the same file type and are valid "TIFF" files. However, they are not considered valid for BIDS.
In BIDS, they asked us to minimize the number of extensions considered valid to be a "BIDS-compliant" dataset. So the 4 extensions I left there (`png`, `tif`, `ome.tif` and `ome.btf`) are the only ones that are microscopy BIDS compliant.

As of today, we could continue to force-index the non-valid extensions, but eventually we will integrate Microscopy to pybids and they won't work anymore. So, I think it's best to remove them from the possibilities as soon as possible.

Reply:
Thanks for the info. Greatly appreciate it!